octopus-hub-things-example/README.md

```diff
@@ -0,0 +1,184 @@
+secondary schools in Germany. It features a number of sensors, and the ability to connect sensors, inputs, and 
+actuators in a relatively simple way. More information and the source code for the schematics can be found on the 
+<a href="http://fab-lab.eu/octopus/">Fab-Lab.eu Octopus page</a>. 
+While Guido Burger offers some boards in his Tindie store, overall the Octopus board is a not-for-profit activity.
+The Octopus board is a small board based on a ESP8266 Chip that can be programmed e.g. using the Arduino IDE.
+The special thing about this board is, that it already contains components that makes it a ready-to-run starter-kit 
+to experiment with the Internet of Things.
+Here you can see a detailed description of the official Octopus Board used for the "Nationaler IT Gipfel".
+
+<img alt="Drawing" style="width: 640px;height: 360px" src="https://cdn.tindiemedia.com/images/resize/tPhVtjN_DuxDKaRkY0-bOKacDbU=/p/74x59:1024x692/full-fit-in/2400x1600/i/52962/products/2016-12-08T20%3A25%3A54.337Z-octo_pinout_final_top.jpg"/>
+
+Source: [Tindimedia](https://cdn.tindiemedia.com/images/resize/tPhVtjN_DuxDKaRkY0-bOKacDbU=/p/74x59:1024x692/full-fit-in/2400x1600/i/52962/products/2016-12-08T20%3A25%3A54.337Z-octo_pinout_final_top.jpg)
+
+
+For example, the Bosch IoT Suite edition board contains the following sensors:
+* [Bosch Sensortec BNO055](https://www.bosch-sensortec.com/bst/products/all_products/bno055) 
+9-axis absolute orientation sensor – provides acceleration, gyroscope, spatial positioning, magnetic field etc.
+* [Bosch Sensortec BME680](https://www.bosch-sensortec.com/bst/products/all_products/bme680)
+environmental sensor – provides temperature, humidity, air pressure, altitude, air quality
+
+![Bosch IoT Suite Edition](images/octopus.jpg)
+
+In this example we use both of these sensors as well as the WiFi component and the two LEDs.
+## Programming your board
+To program your board you can use the [Arduino IDE](https://www.arduino.cc/en/Main/Software).
+There you can write the code that should be executed on your board and upload it to your board.
+
+### Setup IDE
+1. Download [Arduino IDE](https://www.arduino.cc/en/Main/Software)
+2. Configure [ESP8266 board support](https://learn.adafruit.com/adafruit-feather-huzzah-esp8266/using-arduino-ide#install-the-esp8266-board-package)
+3. Install the following libraries (Sketch > Include Library > Manage Libraries)
+    * [Adafruit Unified Sensor library](https://github.com/adafruit/Adafruit_Sensor)
+    * [Adafruit BME680 library](https://github.com/adafruit/Adafruit_BME680)
+    * [Adafruit BNO055 library](https://github.com/adafruit/Adafruit_BNO055)
+    * [Adafruit NeoPixel library](https://github.com/adafruit/Adafruit_NeoPixel)
+    * [PubSubClient library](https://github.com/knolleary/pubsubclient)
+    * [ArduinoJson library](https://github.com/bblanchon/ArduinoJson)
+4. Edit the file `${ArduinoDirectory}/libraries/pubsubclient/src/PubSubClient.h` and set the MQTT_MAX_PACKET_SIZE
```

Comment:
As discussed this has to be done, because with the default compiler settings of the Arduino IDE the constant definitions are ignored when compiling libraries. A info regarding this problem will be added to the README